addons/website/models/website.py

```diff
@@ -339,7 +339,7 @@ def configurator_init(self):
     def configurator_recommended_themes(self, industry_id, palette):
         domain = [('name', '=like', 'theme%'), ('name', 'not in', ['theme_default', 'theme_common'])]
         client_themes = request.env['ir.module.module'].search(domain).mapped('name')
-        client_themes_img = dict([(t, http.addons_manifest[t].get('images_preview_theme', {})) for t in client_themes])
+        client_themes_img = {t: get_manifest(t).get('images_preview_theme', {}) for t in client_themes}
```

Comment:
This key is always present
```suggestion
        client_themes_img = {t: get_manifest(t)['images_preview_theme'] for t in client_themes}
```

Reply:
In case the theme does not exist it would return an empty dict. `get` is safer.